src/lib/stitching/kaws/stitching.ts

```diff
@@ -102,10 +103,19 @@ export const kawsStitchingEnvironmentV2 = (
   const filterArtworksArgsV2 = pageable(filterArtworksArgsV2WithoutPageable)
   return {
     // The SDL used to declare how to stitch an object
-    extensionSchema: `
+    extensionSchema: gql`
     extend type Artist {
       marketingCollections(size: Int): [MarketingCollection]
     }
+    extend type Viewer {
```

Comment:
`marketingCollections` are already available at the root level! But they need to be exposed under `Viewer` (since it's being used in a relay context!). Other fields get automatically exposed under `Viewer` if they're at the root, but I'm guessing that there's something special about stitched fields preventing that?

Reply:
Sorry my bad! I thought this was to power the old-style "feature" pages which weren't Relay-based.

Yea, the automatic exposure in `Viewer` is handled custom here: https://github.com/artsy/metaphysics/blob/9af073c7566d15e695e0a19a70efec2065086f82/src/schema/v2/schema.ts#L148-L151